README.md

```diff
@@ -0,0 +1,44 @@
+# example-approver-policy-plugin
```

Comment:
@SgtCoDFish added the cert-manager badge to the top of all the other repos README files
Suggest we do the same here.

Reply:
Thanks Richard, that's a good idea, I've added it.